services/ui-src/src/components/export/ExportedReportFieldTable.tsx

```diff
@@ -44,23 +45,48 @@ export const ExportedReportFieldTable = ({ section }: Props) => {
   const reportType = report?.reportType as ReportType;
   const hideHintText = reportType === ReportType.MLR;
 
+  // handle ILOS rendering logic
+  const renderIlosVerbiage = () => {
+    const hasIlos =
+      report?.fieldData["ilos"] || report?.fieldData["ilos"]?.length > 0;
```

Comment:
Maybe could simplify this by returning false right away if`report?.fieldData["ilos"]` condition isn't met. Something like
```
if (!report?.fieldData["ilos"]) {
  return false;
}

return section.path === "/mcpar/plan-level-indicators/ilos";
```